tss-esapi/tests/abstraction_ak_tests.rs

```diff
@@ -162,14 +162,14 @@ fn test_create_custom_ak() {
         &mut context,
         ek_rsa,
         HashingAlgorithm::Sha256,
-        SignatureScheme::RsaPss,
+        SignatureSchemeAlgorithm::RsaPss,
         Some(&ak_auth),
         None,
     )
     .unwrap();
 
     assert_eq!(
-        att_key_without.out_public.publicArea.objectAttributes
+        att_key_without.out_public.object_attributes().0
```

Comment:
Think it would be nice to implement a getter for that value stored in `ObjectAttributes`, just so it's more easy to understand what's going on. We could just throw it in a new issue, I guess

Reply:
Hrrm there should be an Into implementation ... for ObjectAttributes.

Reply:
There is ... I will change this.

Reply:
Fixed